temporalcli/commands.gen.go

```diff
@@ -1556,6 +1557,7 @@ func NewTemporalTaskQueueDescribeCommand(cctx *CommandContext, parent *TemporalT
 	s.TaskQueueTypeLegacy = NewStringEnum([]string{"workflow", "activity"}, "workflow")
 	s.Command.Flags().Var(&s.TaskQueueTypeLegacy, "task-queue-type-legacy", "Task Queue type (legacy mode only). Accepted values: workflow, activity.")
 	s.Command.Flags().IntVar(&s.PartitionsLegacy, "partitions-legacy", 1, "Query for all partitions up to this number (experimental+temporary feature) (legacy mode only).")
+	s.Command.Flags().BoolVar(&s.ReportStats, "report-stats", false, "Display task queue statistics.")
```

Comment:
what's the default value? I'm thinking it should be true by default.

Reply:
hmm, why do you think it should be true? The default value is false and I think it should be kept that way - noticed other flags were also following a similar principle unless we have something specific for this one